spec/js/contact_form_element_spec.ts

```diff
@@ -0,0 +1,82 @@
+/**
+ * @jest-environment jsdom
+ */
+
+import ContactUsFormElement from '../../assets/js/contact_us_form_element';
+
+describe('ContactUsFormElement', () => {
+  const now = '2023-01-20T05:00:00Z';
+
+  beforeAll(() => {
+    jest
+      .spyOn(ContactUsFormElement.prototype, 'now', 'get')
+      .mockImplementation(() => new Date(now));
+  });
```

Comment:
I was going to revise this to use [Jest fake timers](https://jestjs.io/docs/jest-object#fake-timers) but ran into some strange issues where the component's `connectedCallback` didn't appear to be called in all cases. I notice we're several versions behind, but the upgrade was also not straightforward. I'm also contemplating how we reconcile the use of Jest in this project with Mocha in the IdP, and whether we migrate toward Mocha instead.

For now, I'll just leave this implementation as-is since it works fine enough.

Attempted revision:

```suggestion
jest.useFakeTimers({ now: new Date('2023-01-20T05:00:00Z') });

describe('ContactUsFormElement', () => {
```